### ISSUE TITLE
Catalina 10.15.7 Support?

### ISSUE THREAD

**Issue by TheBigHoopla** [HuniePotHeads/HuniePop-x64#1]
Hey I was trying to troubleshoot your method to run the game for Mac OS and was having trouble getting the game to run. I saw that you had only posted a successful go with Catalina 10.15.5 and was wondering if that was the issue I was having to get the game to run. I merged the files like you said and moved the game file from the steam common apps to my desktop and renamed it but to no avail. Any help would be appreciated- thanks

**Comment by TheLounger** [comment]
As stated in the ReadMe, the Steam version is not supported. Moving it out of the Steam directory won't help in this case.
The reason I added the part about renaming/moving is because macOS *may* remember/cache some info about an app, such as the fact that it's 32bit and doesn't run. In these cases, renaming or moving an app will make macOS forget about those details and try to start the app again.

Here's the problem with the Steam version:
Upon starting the game, no matter *where* you put it, the app's included SteamWorks logic sees that it's not authenticated with Steam, so then it will launch Steam for authentication *and* file consistency. If it can authenticate, it will attempt to start the game, but not before it has made sure you have the original (32bit) Steam files.

Simply put, the reason that the Steam version isn't supported is because it would require disabling/removing the Steam authentication from the game app.

I will give it try on 10.15.7 and get back to you when I got that up and running, soon-ish.

**Comment by TheLounger** [comment]
Just tested HuniePop (non-Steam version) on macOS Catalina 10.15.7 and it's working without issues.

![HuniePop-x64 macOS 10.15.7](https://user-images.githubusercontent.com/4087076/98502803-39a9ee80-2253-11eb-945b-e4c5d3449b75.png)

The only additional fix someone might need to apply, is making the HuniePop binary executable using the terminal:
```shell
chmod u+x HuniePop.app/Contents/MacOS/HuniePop
```
I will add this to the ReadMe.
